BLSQ/iaso
# IA-2556: Webpack dev server upgrade

use newer version of Webpack dev server to improve dev

Related JIRA tickets : IA-2556

## Self proofreading checklist


## Changes

- upgrade webpack-bundle-tracker to 1.0.0-alpha.1
- patch django-webpack-loader version 1.3.0 [here](https://github.com/BLSQ/django-webpack-loader/pull/1) 
- use patched version of webpack loader
- modify script to launch dev server

## How to test

Reinstall front and back and launch dev server. Check that hot reload is working

## Notes

[This](https://github.com/BLSQ/django-webpack-loader/pull/1) should be merged to master and we can use the new commit hash in requirement.txt
